samples/student-manager/pom.xml

```diff
@@ -16,7 +16,7 @@
 
     <properties>
         <student.manager.version>4.4.37-SNAPSHOT</student.manager.version>
-        <carbon.p2.plugin.version>1.5.4</carbon.p2.plugin.version>
+        <carbon.p2.plugin.version>1.6.1-SNAPSHOT</carbon.p2.plugin.version> <!-- maven-tools branch: master_java10 -->
```

Comment:
Changing to SNAPSHOT is acceptable here as this is still work-in-progress. 
(I am putting this comment to clear any doubt why this is changed as such) 
This needs to be properly versioned after p2 plugin released.